front/app/containers/InitiativesShowPage/index.tsx

```diff
@@ -29,6 +29,12 @@ const StyledInitiativeShowPageTopBar = styled(InitiativeShowPageTopBar)`
   z-index: 1000;
 `;
 
+const StyledInitiativesShow = styled(InitiativesShow)`
+  ${({ theme }) => media.tablet`
+    margin-top: ${theme.menuHeight}px;
+  `}
```

Comment:
Page was suddenly hiding under the nav on mobile. Couldn't figure out how that was happening but fixed :)